l1-contracts/bootstrap.sh

```diff
@@ -38,6 +38,11 @@ function build {
       --optimize \
       --optimizer-runs 200
 
+    # Step 3: Test the Honk ZK verifier (temporary)
+    forge test --match-contract Add2HonkZKTest --match-test testValidProof
```

Comment:
small request to add this under function test (callable with l1-contracts/bootstrap.sh test)

Reply:
made a proper flow test so will remove these